File: Source/DroolTool.Models/DataTransferObjects/DroolWatershedMetricDto.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace DroolTool.Models.DataTransferObjects
{
    public class DroolWatershedMetricDto
    {
        public int? MetricYear { get; set; }
        public int? MetricMonth { get; set; }
        public string TotalMonthlyDrool { get; set; }
        public string OverallParticipation { get; set; }
        public string PercentParticipation { get; set; }
    }
}

File: Source/DroolTool.Web/src/app/shared/models/drool-watershed-metric-dto.ts
export class DroolWatershedMetricDto{
    public MetricYear: number;
    public MetricMonth: number;
    public TotalMonthlyDrool: string;
    public OverallParticipation: string;
    public PercentParticipation: string;

    constructor(obj?: any){
        Object.assign(this, obj);
    }
}